labs/FineTuning/HuggingFaceExample/02_inference/Inference-TinyLlama-1.1B.ipynb

```diff
@@ -0,0 +1,1467 @@
+    "<|assistant|>\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "3d0021ab-8428-4851-88f3-882a75d702e6",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "<|system|>\n",
+      "You are an text to SQL query translator. Users will ask you questions in English and you will generate a SQL query based on the provided SCHEMA.\n",
+      "SCHEMA:\n",
+      "CREATE TABLE courses (course_name VARCHAR, course_id VARCHAR); CREATE TABLE \n",
+      "student_course_registrations (student_id VARCHAR, course_id VARCHAR)</s>\n",
+      "<|user|>\n",
+      "What are the ids of all students for courses and what are the names of those courses?</s>\n",
+      "<|assistant|>\n",
+      "SELECT student_id FROM student_course_registrations JOIN courses ON student_course_registrations.course_id = courses.course_id WHERE courses.course_name = \"Courses\";</s>\n"
+     ]
+    }
+   ],
+   "source": [
+    "import json\n",
+    "result = predictor.predict(\n",
+    "    {\"inputs\": example, \"parameters\": {\"do_sample\": True,\"max_new_tokens\": 100,\"temperature\": 0.7,\"watermark\": True}}\n",
+    ")\n",
+    "\n",
+    "print(result[0]['generated_text'])\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "9e35f617-4721-419d-8d23-78fb6dee5d50",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "example=\"\"\"\n",
+    "<|system|>\n",
+    "You are an text to SQL query translator. Users will ask you questions in English and you will generate a SQL query based on the provided SCHEMA.\n",
+    "SCHEMA:\n",
+    "CREATE TABLE table_name_9 (wins INTEGER, year VARCHAR, team VARCHAR, points VARCHAR)</s>\n",
+    "<|user|>\n",
+    "Which highest wins number had Kawasaki as a team, 95 points, and a year prior to 1981?</s>\n",
+    "<|assistant|>\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "91c03b95-81c4-4907-8e22-cf15e385b920",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "<|system|>\n",
+      "You are an text to SQL query translator. Users will ask you questions in English and you will generate a SQL query based on the provided SCHEMA.\n",
+      "SCHEMA:\n",
+      "CREATE TABLE table_name_9 (wins INTEGER, year VARCHAR, team VARCHAR, points VARCHAR)</s>\n",
+      "<|user|>\n",
+      "Which highest wins number had Kawasaki as a team, 95 points, and a year prior to 1981?</s>\n",
+      "<|assistant|>\n",
+      "SELECT MAX(wins) FROM table_name_9 WHERE team = \"kawasaki\" AND points = \"95\" AND year < 1981;</s>\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = predictor.predict(\n",
```

Comment:
Do we know how well this model addresses these questions? Would be great to know if fine-tuning is helping us in the case.